dist/template.html

```diff
@@ -0,0 +1,44 @@
+<html>
+  <head>
```

Comment:
what is the difference between this file a dist/index.html?

Reply:
dist/index.html is just the default file that is loaded (with restaurant id=305) when the client requests '/'.  However, template.html is the file that is used by the server to inject the requested restaurant id (GET '/3267') into the HTML file (and subsequently to the React components) to return to the client.